apps/hls_examples/hls_support/HalideRuntimeZynq.cpp

```diff
@@ -242,6 +242,23 @@ int halide_zynq_hwacc_sync(int task_id){
     return res;
 }
 
+void buffer_to_stencil(struct halide_buffer_t* image, struct UBuffer* stencil) {
+    size_t nDims = image->dimensions;
+    if (nDims < 2) {
+        printf("buffer_t has less than 2 dimension, not supported in CMA driver.");
+    }
+    stencil->width = image->dim[nDims - 2].extent; // length of the array
+    stencil->height = image->dim[nDims - 1].extent;
+    stencil->depth = (image->type.bits + 7) / 8;
```

Comment:
It is not the same as zynq.cpp. Is it intended?

Reply:
It seems that I forgot to change back some edits I made for testing. I will fix that.